tools/workspace-plugin/src/executors/generate-api/executor.ts

```diff
@@ -46,7 +46,8 @@ function normalizeOptions(schema: GenerateApiExecutorSchema, context: ExecutorCo
 
   const project = context.projectsConfigurations!.projects[context.projectName!];
 
-  const resolveLocalFlag = Boolean(process.env.__FORCE_API_MD_UPDATE__) || isCI() ? false : resolvedSchema.local;
+  const resolveLocalFlag = Boolean(process.env.__FORCE_API_MD_UPDATE__) || (isCI() ? false : resolvedSchema.local);
```

Comment:
this was incorrectly resolving local when env was set -> missing explicit `()` to group the expression